web/router.ex

```diff
@@ -35,6 +35,11 @@ defmodule Healthlocker.Router do
     put "/account/update", AccountController, :update
     get "/account/consent", AccountController, :consent
     put "/account/update-consent", AccountController, :update_consent
+    get "/account/security", AccountController, :security
+    get "/account/edit-security", AccountController, :edit_security
```

Comment:
these urls aren't very REST-ful. we could change them to `/account/security/edit`